client/src/main/scala/drt/client/components/StatusPage.scala

```diff
@@ -27,7 +27,10 @@ object StatusPage {
           <.h2("Feeds status"),
           feedStatusesPot.render((allFeedStatuses: Seq[FeedSourceStatuses]) => {
             val isLiveFeedAvailable = allFeedStatuses.count(_.feedSource.name == LiveFeedSource.name) > 0
-            allFeedStatuses.map(feed => {
+
+            val  allFeedStatusesSeq = allFeedStatuses.filter(_.feedSource.name == ApiFeedSource.name) ++ allFeedStatuses.filterNot(_.feedSource.name == ApiFeedSource.name)
```

Comment:
is `.name` necessary here? `_.feedSource == ApiFeedSource` should work right?

Reply:
sure , i think , that should work . i just field comparison is better than object .